source/getting-started/concepts-terminology.markdown

```diff
@@ -20,7 +20,10 @@ For a full list of compatible integrations, refer to the [integrations](/integra
 
 Once an integration has been added, the hardware and/or data are represented in Home Assistant as [devices and entities](#devices--entities).
 
-## Devices & entities
+## Entities
+Entities are the base building block to hold data in home assistant. Examples are sensors, input and output peripherals, and non-physical data records. Entities have one or more of numeric values, states and other components. It may be helpful to review specific types of entities in the [developer's reference](https://developers.home-assistant.io/docs/core/entity)
```

Comment:
Good point. I like the idea of introducing the term entity first. In the meantime, we now also have a definition of the term in the glossary. I added a link there and reused the introduction from that definition. Let me know what you think.

I removed the reference to the developer documentation. The user documentation should contain all the information that is relevant for the user, while not providing all the details that are required only for developping. If there is missing information in the user docs, we should add this component to the user docs, rather than pointing to the developer docs. It is tempting to link to the dev docs, I'm sure I have done it before.

Reply:
All good. Thank you for the improvements and accepting this.